cloud/sts.go

```diff
@@ -19,6 +19,8 @@ import (
 type STSManager interface {
 	// AssumeRole gets the credentials for a role as the given task.
 	AssumeRole(ctx context.Context, taskID string, opts AssumeRoleOptions) (AssumeRoleCredentials, error)
+	// GetCallerIdentity gets the credentials for a role as the given task.
+	GetCallerIdentity(ctx context.Context) (string, error)
```

Comment:
I think it's a mistake that this description is the same as AssumeRole, especially since it doens't take in a taskID. Can you re-clarify? How does this function differ from AssumeRole?

Reply:
Mentioned above but renamed to say the Caller Identity's ARN